tests/testthat/test-cube-dims.R

```diff
@@ -45,32 +63,32 @@ with_mock_crunch({
         expect_identical(aliases(book[[1]]), c("total", "allpets", "q1"))
         expect_identical(descriptions(book[[1]]),
             c("",
-            "Do you have any of these animals as pets? Please select all that apply.",
-            "What is your favorite pet?"))
+                "Do you have any of these animals as pets? Please select all that apply.",
+                "What is your favorite pet?"))
     })
     test_that("Getting those attributes from TabBookResult (the row/sheet variables)", {
         expect_identical(names(book)[2:4],
             c("Pet", "Pets by location", "Number of dogs"))
         expect_identical(aliases(book)[2:4], c("q1", "petloc", "ndogs"))
         expect_identical(descriptions(book)[2:4],
             c("What is your favorite pet?",
-            "Name the kinds of pets you have at these locations.",
-            ""))
+                "Name the kinds of pets you have at these locations.",
+                ""))
     })
 
     test_that("subvariable/subrefrence methods", {
         catarray_x_mr <- loadCube(test_path("cubes/catarray-x-mr.json"))
         cat_array_var <- variables(catarray_x_mr)[[1]]
         expect_equal(aliases(subvariables(cat_array_var)),
-                     c("cat_feeling", "dog_feeling"))
+            c("cat_feeling", "dog_feeling"))
         expect_equal(names(subvariables(cat_array_var)),
-                     c("cat_feeling", "dog_feeling"))
+            c("cat_feeling", "dog_feeling"))
 
         mr_var <- variables(catarray_x_mr)[[3]]
         expect_equal(aliases(subvariables(mr_var)),
-                     c("food_opinion#", "rest_opinion#", "play_opinion#"))
+            c("food_opinion#", "rest_opinion#", "play_opinion#"))
         expect_equal(names(subvariables(mr_var)),
-                     c("food_opinion", "rest_opinion", "play_opinion"))
+            c("food_opinion", "rest_opinion", "play_opinion"))
     })
```

Comment:
I thought we had settled on using the [tidy style](http://style.tidyverse.org/functions.html#long-lines-1) and RStudio's default of "Vertically align arguments in auto-indent" of aligning with the opening `(` on subsequent lines, but I might be wrong about that. I'm happy to turn that off in my IDE if we want to always go to the next indent point and not align with `(` like you have here.

Reply:
I'm deferring this change for now to make the diffs here as clean as possible. I'm happy to re-align this file before merging to what we decided.